source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -181,6 +181,19 @@ If the decommission fails, customers should open a |SUBNET| issue for further as
 Community users without a SUBNET subscription can retry the decommission process or seek additional support through the `MinIO Community Slack <https://slack.min.io/>`__.
 MinIO provides Community Support at best-effort only and provides no :abbr:`SLA <Service Level Agreement>` around responsiveness.
 
+Decommissioning a Server with Tiering Enabled
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionchanged:: RELEASE.2023-03-20T20-16-18Z
+
+For deployments with tiering enabled and active in one or more buckets, decommissioning proceeds as expected.
+Objects subject to tiering rules move to an active pool as expected.
+
+.. note:: 
+
+   For older versions, active tiering prevents decommissioning from starting. 
+   Upgrade to RELEASE.2023-03-20T20-16-18Z or later to decommission pools where tiering is active.
```

Comment:
As expected leverages an assumption of how this should work.

Spitballing

```suggestion
.. versionchanged:: RELEASE.2023-03-20T20-16-18Z

   For deployments with tiering enabled and active, decommissioning moves the object references to a new active pool.
   Applications can continue issuing GET requests against those objects where MinIO handles transparently retrieving them from the remote tier.

In older MinIO versions, tiering configurations prevent decommissioning. 
You must upgrade to enable decommissioning with tiering configurations
```

Not sure the last sentence is necessary. Also don't need the note.

Reply:
Concur. As someone just getting started with this, the additional context is the difference between "does this mean what I think it does?" and "oh right I know what this is about." Yay.

Reply:
I like it. Done!